35.GeometryCreator/main.cpp

```diff
@@ -288,7 +293,14 @@ class GeometryCreatorSampleApp : public ApplicationBase
 
 		auto createSpecializedShaderFromSourceWithIncludes = [&](const char* source, asset::IShader::E_SHADER_STAGE stage, const char* origFilepath)
 		{
-			auto resolved_includes = assetManager->getGLSLCompiler()->resolveIncludeDirectives(source, stage, origFilepath);
+			auto computeUnspec = core::make_smart_refctd_ptr<asset::ICPUShader>(source, stage, asset::IShader::E_CONTENT_TYPE::ECT_GLSL);
+			auto shaderCompiler = assetManager->getCompilerSet()->getShaderCompiler(asset::IShader::E_CONTENT_TYPE::ECT_GLSL)->getDefaultIncludeFinder();
+			shaderCompiler->addSearchPath(origFilepath, core::make_smart_refctd_ptr<asset::IShaderCompiler::CFileSystemIncludeLoader>(core::smart_refctd_ptr(system)));
```

Comment:
I think we talked about this and that you shouldn't do it. no need to addSearchPaths

Reply:
fixed it myself